app/src/main/AndroidManifest.xml

```diff
@@ -1,27 +1,38 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.dgaf.happyhour">
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.dgaf.happyhour" >
+
     <uses-sdk android:minSdkVersion="19" />
+
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="com.dgaf.happyhour.permission.MAPS_RECEIVE" />
     <uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+
     <uses-feature
         android:glEsVersion="0x00020000"
         android:required="true" >
     </uses-feature>
 
+    <android:uses-permission android:name="android.permission.READ_PHONE_STATE" />
+    <android:uses-permission
+        android:name="android.permission.READ_EXTERNAL_STORAGE"
+        android:maxSdkVersion="18" />
+
     <application
-        android:name="Burrd"
+        android:name=".Burrd"
```

Comment:
Was this so you could have the app store app and the dev version installed at the same time?